source/exe/BUILD

```diff
@@ -67,11 +67,11 @@ envoy_cc_library(
 )
 
 # Non-core HTTP/TCP proxy libraries.
+# TODO(mattklein123): Delete this when extension migration is complete.
```

Comment:
@htuch @alyssawilk @jmarantz FYI for Google import. I think you should be able to just overwrite the envoy_all_extensions() to nuke extensions you don't want?

Reply:
Sure, this is pretty neat.